website/docs/configuration/overview.mdx

```diff
@@ -43,6 +43,19 @@ Or with npx:
 npx -- @finos/git-proxy --config ./config.json
 ```
 
+### Change ports
+By default, Git Proxy uses port 8000 to expose the Git Server and 8080 for the UI frontend; 
+you can change ports by setting the `GIT_PROXY_SERVER_PORT` and `GIT_PROXY_UI_PORT` 
```

Comment:
[optional]

```suggestion
The ports can be changed by setting the `GIT_PROXY_SERVER_PORT` and `GIT_PROXY_UI_PORT` 
```

IMO: For documentation it is best to keep the language impersonal and objective..

so no `I`, `me` or `you`